src/components/ResultsItemDetails.vue

```diff
@@ -4,88 +4,60 @@
       <button @click="savePark" class='button-save-park'>SAVE</button>
```

Comment:
This button still needs to save the whole result object. Will put this on the to-do list